lib/mayaUsd/ufe/UsdObject3d.cpp

```diff
@@ -84,8 +84,12 @@ Ufe::BBox3d UsdObject3d::boundingBox() const
     // Add maya-specific extents
     UsdMayaUtil::AddMayaExtents(bbox, fPrim, time);
 
-    auto range = bbox.ComputeAlignedRange();
-    return Ufe::BBox3d(toVector3d(range.GetMin()), toVector3d(range.GetMax()));
+    auto        range = bbox.ComputeAlignedRange();
+    Ufe::BBox3d ufeBBox(toVector3d(range.GetMin()), toVector3d(range.GetMax()));
+
+    Ufe::BBox3d pulledBBox = getPulledPrimsBoundingBox(path);
+
+    return combineUfeBBox(ufeBBox, pulledBBox);
```

Comment:
Thanks for putting the combineUfeBBox in the new UsdUfe library. Just for your information I'm currently working on moving Object3d to Usd Ufe. So I will have to adapt to these new changes, especially the calling of the new helper function `getPulledPrimsBoundingBox` (which is in MayaUsd).